vignettes/tidytranscriptomics.Rmd

```diff
@@ -896,6 +896,11 @@ markers_df %>%
     custom_theme
 ```
 
+## Key Points
+- `tidybulk` allows streamlined multi-method analyses
+- The function `pivot_longer` from `tidyr` is useful to operate serially across consistent portions of data for comparative purposes. For example, the statistics for several methods. 
```

Comment:
pivot_longer is not used in Part 2, it's used in Part 1. Do you mean to say nest from tidyr? Would it make sense instead to mention the map functions you're making use of from purrr?

Reply:
I meant pivot_ as it can help to format all methods and all statistcis at the same time, for then nesting it. and/or plotting with facets

Reply:
ok but pivot_longer is not shown in Part 2? these should be the key points  or what has been shown in this part so I think either need to show what you mean with pivot_longer in part 2 or drop it from the key points here.

Reply:
Ah ok :) happy to drop it